benchmark/opperf/utils/op_registry_utils.py

```diff
@@ -223,13 +226,17 @@ def get_all_random_sampling_operators():
     -------
     {"operator_name": {"has_backward", "nd_op_handle", "params"}}
     """
+    # Additional Random Sampling ops which do not start with "random_" or "sample_"
+    additional_random_sampling_ops = ['GridGenerator', 'BilinearSampler']
+
     # Get all mxnet operators
     mx_operators = _get_all_mxnet_operators()
 
     # Filter for Random Sampling operators
     random_sampling_mx_operators = {}
     for op_name, op_params in mx_operators.items():
-        if op_name.startswith(("random_", "sample_")) and op_name not in unique_ops:
+        if (op_name.startswith(("random_", "sample_")) or \
+            op_name in additional_random_sampling_ops) and op_name not in unique_ops:
```

Comment:
lets get rid of this unique_ops condition since there are no longer unique_ops remaining!